packages/expo-local-authentication/ios/EXLocalAuthentication/EXLocalAuthentication.m

```diff
@@ -137,4 +157,20 @@ static BOOL EXIsFaceIDDevice()
   return isFaceIDDevice;
 }
 
+static BOOL EXIsTouchIDDevice()
+{
+  static BOOL isTouchIDDevice = NO;
+  if (@available(iOS 11.0, *)) {
```

Comment:
`@available` shouldn't be there or at least not with iOS 11.0, because TouchID is available in iOS 10.0 too